pkg/clusteragent/admission/controllers/webhook/common.go

```diff
@@ -18,7 +18,9 @@ import (
 func buildLabelSelectors(useNamespaceSelector bool) (namespaceSelector, objectSelector *metav1.LabelSelector) {
 	var labelSelector metav1.LabelSelector
 
-	if config.Datadog.GetBool("admission_controller.mutate_unlabelled") {
+	if config.Datadog.GetBool("admission_controller.mutate_unlabelled") ||
```

Comment:
This will now subtly permit anything else using `mutate_unlabelled` to run if APM is enabled. I did a quick check and it looks like only APM functionality is using `buildLabelSelectors`. 

Since this function is in `common` I think we should rename the function to `buildAPMLabelSelectors` or similar so that if other functionality added in the future tries to use `mutate_unlabelled` it doesn't mistakenly get enabled if APM instrumentation is.

Reply:
This logic is just limiting a number of pods that Admission Controller will be looking at to potentially mutate them. It's not going to trigger any mutations on its own. So the pods that don't have mutateUnlabelled or APM SSI won't be considered by admission controller unless they have the label set to `true`.

Reply:
oh right 🤦‍♂️ this thing just opts the pods into being filtered out later on. All good here then!